app/database/models/ms_schema/mentorship_relation.py

```diff
@@ -0,0 +1,151 @@
+        mentee_agreed: numeric that indicates the date when mentee accepted to a program.
+    """
+
+    # Specifying database table used for MentorshipRelationModel
+    __tablename__ = "mentorship_relations"
+    __table_args__ = {"schema": "public", "extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    
+    # personal data
+    mentor_id = db.Column(db.Integer, db.ForeignKey("public.users.id"))
+    mentee_id = db.Column(db.Integer, db.ForeignKey("public.users.id"))
+    action_user_id = db.Column(db.Integer, nullable=False)
+    mentor = db.relationship(
+        # UserModel,
+        "UserModel",
+        backref="mentor_relations",
+        primaryjoin="MentorshipRelationModel.mentor_id == UserModel.id",
+    )
+    mentee = db.relationship(
+        # UserModel,
+        "UserModel",
+        backref="mentee_relations",
+        primaryjoin="MentorshipRelationModel.mentee_id == UserModel.id",
+    )
+
+    creation_date = db.Column(db.Numeric("16,6", asdecimal=False), nullable=False)
+    
+    accept_date = db.Column(db.Numeric("16,6", asdecimal=False))
+    
+    start_date = db.Column(db.Numeric("16,6", asdecimal=False)) 
+    end_date = db.Column(db.Numeric("16,6", asdecimal=False)) 
+
+    state = db.Column(db.Enum(MentorshipRelationState), nullable=False)
+    notes = db.Column(db.String(400))
+
+    tasks_list_id = db.Column(db.Integer, db.ForeignKey("public.tasks_list.id"))
+    tasks_list = db.relationship(
+        TasksListModel, uselist=False, backref="mentorship_relation"
+    )
+
+    mentorship_relation_extension = db.relationship(
+        "MentorshipRelationExtensionModel",
+        backref="mentorship_relation",
+        uselist=False,
+        cascade="all,delete",
+        passive_deletes=True,
+    )
+    
+
+    # pass in parameters in a dictionary
+    def __init__(
+        self,
+        action_user_id,
+        mentor_user,
+        mentee_user,
+        creation_date,
+        end_date,
+        state,
+        notes,
+        tasks_list,
+    ):
+
+        self.action_user_id = action_user_id
+        self.mentor = mentor_user
+        self.mentee = mentee_user  # same as mentee_user.mentee_relations.append(self)
+        self.creation_date = creation_date
+        self.end_date = end_date
+        self.state = state
+        self.notes = notes
+        self.tasks_list = tasks_list
+
+    def json(self):
+        """Returns information of mentorship as a json object."""
+        return {
+            "id": self.id,
+            "action_user_id": self.action_user_id,
+            "mentor_id": self.mentor_id,
+            "mentee_id": self.mentee_id,
+            "creation_date": self.creation_date,
+            "accept_date": self.accept_date,
+            "start_date": self.start_date,
+            "end_date": self.end_date,
+            "state": self.state,
+            "notes": self.notes,
+        }
+
+    # def __repr__(self):
```

Comment:
Delete this if not needed

Reply:
the #def __repr__(self)? Ok. will do that,